injitest/src/main/java/inji/utils/TestDataReader.java

```diff
@@ -77,21 +77,21 @@ private static String getValueForKey(String key) {
             case "fullNameSunbird":
                 return "PolicyTestName";
             case "policyNameSunbird":
-                return "Talapathy Rasigar Mandram";
+                return "PolicyTest";
             case "policyNumberSunbird":
                 return "12072024";
```

Comment:
Can we use policy number with hyphen like `120-720-24`?
Recently we faced the issue with hyphen policy number. Normal number were working fine

Reply:
added policy number with hyphen